config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -99,7 +99,9 @@ periodics:
       - --timeout=120m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20190911-069bf1f-master
   annotations:
-    testgrid-dashboards: google-windows, sig-windows
+    fork-per-release: "true"
+    for-per-release-replacements: "--extract=ci/k8s-master -> --extract=ci/k8s-beta"
```

Comment:
Can we add fork-per-release with something like `--extract=ci/k8s-beta -> --extract=ci-k8s-beta1` for the jobs below?

Maybe search for other test config files with `for-per-release-replacements` for examples to imitate.

Reply:
Look like the config forker only forks a job from master to beta, for stable1, 2, 3 may need manually update. @yujuhong @justaugustus Is my understanding correct?

Reply:
This should be `fork-per-release-replacements` - note the `k`.

The [config-rotator](https://github.com/kubernetes/test-infra/tree/master/experiment/config-rotator) handles the subsequent moves. by replacing `beta` -> `stable1` -> `stable2` -> `stable3` as jobs get older.

It looks like you can move the `ci-kubernetes-e2e-windows-gce-k8s{beta,stable1,stable2}` jobs below into the 1.16/1.15/1.14 files as appropriate and rely on the tooling to update/remove them as appropriate, as well as create the new one for 1.17 using these annotations when that rolls around.

Reply:
Thanks @Katharine Created 1.16/1.15/1.14.yaml files, referring `config/jobs/kubernetes/sig-release/release-branch-jobs`.  
I'm new to this project, could you please guide me further about what value should I put for `fork-per-release-periodic-interval`, `fork-per-release-cron` or `fork-per-release-generic-suffix`?

Reply:
Sorry, I was unclear - I was specifically suggesting moving them to the ones [under sig-release](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/sig-release/release-branch-jobs) - that's where automatically generated release-branch config lives, and the only place it's automatically managed.

Looking at your jobs, you probably want configuration like this (on the primary job only):

```yaml
annotations:
  fork-per-release: "true" # enables automatic forking
  fork-per-release-replacements: "--extract=ci/k8s-master -> --extract=ci/k8s-beta" 
```

Which is basically what you already have. You might also want `fork-per-release-generic-suffix: "true"`, which will cause the older jobs to be named `ci-kubernetes-e2e-windows-gce-beta` instead of `ci-kubernetes-e2e-windows-gce-1-16`, etc. The intervals are always the same, so you don't need to specify `fork-per-release-periodic-interval` or `fork-per-release-cron`.

What you _can't_ do is maintain the current naming scheme: you must use either `-beta` or `-1-16` type suffixes (depending on `fork-per-release-generic-suffix`). There is no support for `-k8sbeta` type suffixes. I would like to see confirmation from people who might care (presumably sig-windows, possibly sig-release) that this is okay.

Reply:
Thanks, it's very helpful. Changed `-k8sbeta` to `-k8s-beta` for preview. Seems we could remove the `-k8s`, but need confirmation.